apiary.apib

```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
+
+<center><b>Example Payload</b></center>
+
+    {
+     "businessNumbers": [
+       145308764185002,
+       145308769721320
+      ],
+     "sourceAlias": [
+     "google","citysearch"
+     ],
+     "employees": [
+       {
+      "firstName": "String",
+      "lastName": "String",
+      "phone": "String"
+       }
+     ],
+     "fromDate": "04/01/2013",
+     "toDate": "05/3/2015"
+   }
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Request Payload</caption>
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>fromDate</td>
+            <td>Review date (start) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>toDate</td>
+            <td>Review date (end) e.g. "04/01/2013"</td>
+        </tr>
+        <tr>
+            <td>sourceAlias</td>
+            <td>Aggregation source name e.g. "google","citysearch" etc. Note: use "our_website" to filter reviews posted on Birdeye platform or use "direct_feedback" for direct feedback. </td>
+        </tr>
+         <tr>
+            <td>employees</td>
+            <td>Employee details eg. FirstName,LastName and phone.</td>
+        </tr>
+        <tr>
+            <td>businessNumbers</td>
+            <td>businessNumbers under enterprise account.</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+#### Response
+
+<center><b>Example Response</b></center>
+
+  {
+  "employees": [
+    {
+      "employee": {
+        "firstName": "rishab",
+        "lastName": "singh",
+        "phone": "4424242424"
+      },
+      "countByRating": [
+        {
+          "rating": 0,
+          "count": 34
+        },
+        {
+          "rating": 1,
+          "count": 22
+        },
+        {
+          "rating": 2,
+          "count": 33
+        },
+        {
+          "rating": 3,
+          "count": 44
+        },
+        {
+          "rating": 4,
+          "count": 44
+        },
+        {
+          "rating": 5,
+          "count": 56
+        }
+      ]
+    }
+  ]
+}
+
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Response Body</caption>
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>employee</td>
+            <td>Employee details eg. FirstName,LastName and phone.</td>
+        </tr>
+         <tr>
+            <td>countByRating</td>
+            <td>Review's count against the rating</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+For facebook sourcetype, following table can be used to check recommendations.
```

Comment:
**sourceAlias** should be used instead of sourceType for uniform usage of keywords

Reply:
Done